attributes/knotx.rb

```diff
@@ -54,6 +54,8 @@
 default['knotx']['debug_enabled'] = false
 default['knotx']['debug_port'] = '28092'
 
+default['knotx']['instance_opts'] = ''
```

Comment:
Are we certain we want to leave the default attribute empty?

Reply:
Yes, we want it to be empty by default.
In that way `run-knotx` command will not get any attributes and Knot.x instance will start using default settings (`$KNOTX_HOME/conf/bootstrap.json` file).